data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -73,6 +74,7 @@ protected DeleteFilter(
       boolean needRowPosCol) {
     this.filePath = filePath;
     this.counter = counter;
+    this.requestedSchema = requestedSchema;
```

Comment:
Can we be consistent with the name? `requestedSchema` or `expectedSchema`? I guess `expectedSchema` is more commonly used.

Reply:
Changed to `expectedSchema`